components/Tabs/Tabs.js

```diff
@@ -1,63 +1,68 @@
 class TabLink {
-  constructor(tabElement){
+  constructor(tabElement) {
     // assign this.tabElement to the tabElement DOM reference
-    // this.tabElement;
-    
+
+    this.tabElement = tabElement;
+
     // Get the `data-tab` value from this.tabElement and store it here
-    // this.tabData = ; 
-    
-    // We need to find out if a user clicked 'all' cards or a specific category.  Follow the instructions below to accomplish this task:    
-    
-    /* <- Delete this comment block when you work on the if statement
+
+    this.tabData + this.tabElement.dataset.tab;
```

Comment:
Looks like you have a '+' instead of a "="!! 
Just a mis type, no worries :)

Reply:
Fixed, thanks